src/Umbraco.Core/IO/MediaFileSystem.cs

```diff
@@ -68,5 +76,16 @@ public void DeleteThumbnails(string path)
 			GetThumbnails(path)
 				.ForEach(DeleteFile);
 		}
+
+	    public void CopyThumbnails(string sourcePath, string targetPath)

```

Comment:
@zpqrtbnk I don't understand this one... is this now part of IFileSystem? This change seems to require that `MediaFileSystem` is used wherever we use IFileSystem for media which is a breaking change and will not work for already existing IFileSystem implementations (like the Azure media file system)

Also note that the thumbnails we generate will be a thing of the past and ImageProcessor will be responsible for processing thumbnails.